src/Compilers/Core/Portable/Generated/Operations.Generated.cs

```diff
@@ -3345,57 +3347,38 @@ public interface IWithOperation : IOperation
     #endregion
 
     #region Implementations
-    internal abstract partial class BaseBlockOperation : OperationOld, IBlockOperation
+    #nullable enable
+    internal sealed partial class BlockOperation : Operation, IBlockOperation
     {
-        internal BaseBlockOperation(ImmutableArray<ILocalSymbol> locals, SemanticModel semanticModel, SyntaxNode syntax, ITypeSymbol type, ConstantValue constantValue, bool isImplicit)
-            : base(OperationKind.Block, semanticModel, syntax, type, constantValue, isImplicit)
+        private IEnumerable<IOperation>? _lazyChildren;
```

Comment:
Thought the intent of this effort was to remove the laziness of IOperation hence surprised to see this. Is this a transient state?

Reply:
No. We'll still lazily calculate and cache the `Children` property, as it doesn't actually have a risk associated with it like lazily realizing the actual child `Children` will refer to.